src/backend/joanie/core/utils.py

```diff
@@ -1,9 +1,21 @@
 """
 Utils that can be useful throughout Joanie's core app
 """
+import base64
+
 from django.utils.text import slugify
 
+from PIL import Image
+
 
 def normalize_code(code):
     """Normalize object codes to avoid duplicates."""
     return slugify(code, allow_unicode=False).upper() if code else None
+
+
+def image_to_base64(image):
+    """Return the src string of the base64 encoding of an image."""
+    mime_type = Image.open(image).get_format_mimetype()
```

Comment:
As the image is already open is it possible to use this instance to get the `base64` representation of the image ? By saving it in a buffer and encoding then this buffer in base64 ?

Reply:
Good idea. I proposed an improvement inspired by https://github.com/django/django/blob/d0ea01af2814e276c4a090e599d5fd11c6ce4bf8/django/core/files/images.py#L33
WDYT?